Makefile

```diff
@@ -1,40 +1,41 @@
-# You may not be able to build with a modern Xcode, since we still support 10.6.
-# So, if we have Xcode 4.6 around, we should use it.
+# For the building the driver for MacOS 10.11+, the preferred
+# XCode version is 7.3.1.
 #
-# You can do this by downloading Xcode 4.6.3 as a dmg from Apple, then
-# copying the contents into /Applications, then taking your ancient
-# MacOSX10.6.sdk that you keep kicking around, and copying it into
-# /Applications/Xcode-4.6.3.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/. 
-# Don't you love backwards compatibility?
-
-XCODEBUILD_ANCIENT ?= /Applications/Xcode-4.6.3.app/Contents/Developer/usr/bin/xcodebuild
-XCODEBUILD_MODERN ?= xcodebuild
-
-# Ok, no joy there?
-ifeq (,$(wildcard $(XCODEBUILD_ANCIENT)))
-	XCODEBUILD_ANCIENT = xcodebuild
+# You can do this by downloading Xcode 7.3.1 as a dmg from Apple, then
+# copying the contents into /Applications as Xcode-7.3.1.
+# If you also want to build from "release_pre_10_11", you may want to
+# also download "Xcode-4.3.3", and copy "MacOSX10.6.sdk" from it to
+# SDK directory under:
+# /Applications/Xcode-7.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/
+
+XCODE_VER = 7.3.1
+XCODEBUILD ?= $(wildcard /Applications/Xcode*$(XCODE_VER).app/Contents/Developer/usr/bin/xcodebuild)
+
+ifeq (,$(XCODEBUILD))
+    $(error Cannot find Xcode $(XCODE_VER). Please download it from: \
+        "https://developer.apple.com/download" and install as \
+        /Applications/Xcode-$(XCODE_VER)/)
 endif
 
+all: build/Release/HoRNDIS.kext build/_complete
 
-all: build/Release/HoRNDIS.kext build/Release-unsigned/HoRNDIS.kext build/HoRNDIS.pkg
+clean:
+	rm -rf build
 
-# We now sign as part of the xcodebuild process.  Also, 'release' is now
```

Comment:
On second thought, maybe keeping two builds around is a total mess, after all, and we should just point people to the old version for old Macs (and add a "hey, you need to be running 10.11 -- or download an older version!" complaint in the installer if it detects old Macs).

Reply:
I think we shall just say: use versions 9.x for El Capitan or later, and use 8.x for Lion - Yosemite. I've already fixed the installer, so the new code fails if trying to install on the older MacOS: I can add a similar change to the older version in order to prohibit running on the newer MacOS.

Overall, it comes down to cost/benefit tradeoff. It's 2018, and I'm not even sure ANYONE would care about installing the driver on Yosemite or earlier. And even for these people who would, I don't think it would be a major inconvenience to know to download a different version. So, we'd be adding a maintenance cost for almost zero benefit.

Frankly, I'm inclined towards not touching the old code, unless some users come report problems on older MacOS versions.